src/services/score-manager-service.ts

```
@@ -0,0 +1,186 @@
+    player?.setScore(playerScore);
+    this.updateScoreboard();
+    let team: TeamType = TeamType.Red;
+    if (player === this.gameState.getGamePlayer()) {
+      team = TeamType.Blue;
+    }
+    this.showGoalAlert(player, team);
+  }
+
+  public handleRemoteGameOverStartEvent(arrayBuffer: ArrayBuffer | null): void {
+    if (arrayBuffer === null) {
+      return console.warn("Array buffer is null");
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)